### PR TITLE
PS-9468 [DOCS] - Release notes for PS 8.4.3-3

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
new file:   docs/release-notes/8.4.3-3.md
	modified:   docs/release-notes/release-notes-index.md
	modified:   mkdocs-base.yml
	modified:   variables.yml

### CODE REVIEW

File: docs/release-notes/8.4.3-3.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Percona Server for MySQL 8.4.3-3 (2024-12-0)
+
+Get started with [Quickstart Guide for Percona Server for MySQL].
+
+[Percona Server for MySQL 8.4.3-3] includes all the features and bug fixes available in the MySQL 8.4 Community Edition in addition to enterprise-grade features developed by Percona.
```
Comment (patrickbirch) [inline docs/release-notes/8.4.3-3.md]: available in the MySQL 8.4.3

Comment (alina-derkach-oaza) [reply docs/release-notes/8.4.3-3.md]: fixed

**Review by patrickbirch — APPROVED** [review]
Good

**Review by percona-ysorokin — APPROVED** [review]
LGTM

**Pull request merged**